rte/pkg/sysinfo/sysinfo.go

```diff
@@ -86,15 +140,56 @@ func (cfg Config) ToYAMLString() string {
 // NUMA Cell -> deviceIDs
 type PerNUMADevices map[int][]string
 
+// NUMA Cell -> counter
+type PerNUMACounters map[int]int64
+
 type SysInfo struct {
 	CPUs cpuset.CPUSet
 	// resource name -> devices
 	Resources map[string]PerNUMADevices
+	// memory type -> counters
+	Memory map[string]PerNUMACounters
+}
+
+func magnitude(order int) string {
+	orders := []string{
+		"Ki",
+		"Mi",
+		"Gi",
+		"Ti",
+		"Pi",
+		"Ei",
+	}
+	if order < 0 || order > len(orders) {
```

Comment:
it should be ```... || order >= len(orders)``` 
the exact length value is also out-of-bound index.

Reply:
good catch, thanks